webapp/components/EmotionButton/EmotionButton.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="emotion-button">
+    <base-button :id="emotion" circle ghost @click="$emit('toggleEmotion', emotion)">
+      <img class="image" :src="emojiPath" />
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T14:44:09Z" title="Wednesday, January 15th 2020, 3:44:09 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="emotion-button">
+    <base-button :id="emotion" circle ghost @click="$emit('toggleEmotion', emotion)">
+      <img :src="emojiPath" />
```


the pics are not showing up...
![Screenshot from 2020-01-15 15-21-54](https://user-images.githubusercontent.com/26943915/72442543-015d8f00-37ad-11ea-8b13-107293131295.png)

I played around with it, and adding a max width to the `img`, they look good to me 
![Screenshot from 2020-01-15 15-43-02](https://user-images.githubusercontent.com/26943915/72442971-c871ea00-37ad-11ea-86b0-825b8891184b.png)

on hover
![Screenshot from 2020-01-15 15-43-30](https://user-images.githubusercontent.com/26943915/72442982-cc057100-37ad-11ea-9d98-90d08d9044fe.png)

I'll push up the code... you can change, if you have a better solution...

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-22T07:36:46Z" title="Wednesday, January 22nd 2020, 8:36:46 am +01:00">Jan 22, 2020</time>_

---


maybe I should start testing things in Chrome... 😝